network_wrangler/models/roadway/tables.py

```diff
@@ -106,10 +105,10 @@ class Config:
         add_missing_columns = True
         coerce = True
 
-    @pa.dataframe_check
-    def unique_ab(cls, df: pd.DataFrame) -> bool:
-        """Check that combination of A and B are unique."""
-        return ~df[["A", "B"]].duplicated()
```

Comment:
There are duplicate [“A”, “B”] in cc_link, temporarily disabled code; plan to revisit and review later